src/stats/distributions.jl

```diff
@@ -26,49 +26,114 @@ end
 
-@recipe(QQNorm) do scene
-    default_theme(scene, Scatter)
-end
+"""
+    qqplot(x, y; kwargs...)
+Draw a Q-Q plot, comparing quantiles of two distributions. `y` must be a list of
+samples, i.e., `AbstractVector{<:Real}`, whereas `x` can be
+- a list of samples,
+- an abstract distribution, e.g. `Normal(0, 1)`,
+- a distribution type, e.g. `Normal`.
+In the last case, the distribution type is fitted to the data `y`.
+
+The attribute `qqline` (defaults to `:robust`) determines how to compute a fit line for the Q-Q plot.
+Possible values are the following.
+- `:identity` draws the identity line.
+- `:fit` computes a least squares line fit of the quantile pairs.
+- `:robust` computes the line that passes through the first and third quartiles of the distributions.
+- `:none` (or any other value) omits drawing the line.
+Broadly speaking, `qqline = :identity` is useful to see if `x` and `y` follow the same distribution,
+whereas `qqline = :fit` and `qqline = :robust` are useful to see if the distribution of `y` can be
+obtained from the distribution of `x` via an affine transformation.
 
+Graphical attributes are
+- `color` to control color of both line and markers (if `markercolor` is not specified)
+- `linestyle`
+- `linewidth`
+- `markercolor`
+- `strokecolor`
+- `strokewidth`
+- `marker`
+- `markersize`
+"""
 @recipe(QQPlot) do scene
-    default_theme(scene, Scatter)
+    s_theme = default_theme(scene, Scatter)
+    l_theme = default_theme(scene, Lines)
+    Attributes(
+        color = l_theme.color,
+        linestyle = l_theme.linestyle,
+        linewidth = l_theme.linewidth,
+        markercolor = automatic,
+        markersize = s_theme.markersize,
+        strokecolor = s_theme.strokecolor,
+        strokewidth = s_theme.strokewidth,
+        marker = s_theme.marker,
+        inspectable = theme(scene, :inspectable),
+        cycle = [:color],
+    )
 end
 
-convert_arguments(::Type{<:QQNorm}, args...; qqline=:R, kwargs...) =
-    convert_arguments(QQPlot, Distributions.Normal(0, 1), args...; qqline=qqline, kwargs...)
-
-convert_arguments(::Type{<:QQPlot}, args...; kwargs...) =
-    convert_arguments(Scatter, qqbuild(loc(args...)...); kwargs...)
-
-function convert_arguments(P::PlotFunc, h::QQPair; qqline=:identity)
-    line = if qqline in (:fit, :quantile, :identity, :R)
-        xs = [extrema(h.qx)...]
-        if qqline == :identity
-            ys = xs
-        elseif qqline == :fit
-            itc, slp = hcat(fill!(similar(h.qx), 1), h.qx) \ h.qy
-            ys = slp .* xs .+ itc
-        else # if qqline == :quantile || qqline == :R
-            quantx, quanty = quantile(h.qx, [0.25, 0.75]), quantile(h.qy, [0.25, 0.75])
-            slp = diff(quanty) ./ diff(quantx)
-            ys = quanty .+ slp .* (xs .- quantx)
-        end
-        Point{2,Float32}.(xs, ys)
-    else
-        nothing
+"""
+    qqnorm(y; kwargs...)
+
+Shorthand for `qqplot(Normal, y)`. See `qqplot` for more details.
```

Comment:
```suggestion
Shorthand for `qqplot(Normal(0,1), y)`. See `qqplot` for more details.
```

To make it very clear to the users (and later devs updating the code :wink:) that it's the standard normal. cf #1277

Reply:
Good point, thank you!